developer-community/tech-talks.md

```diff
@@ -0,0 +1,240 @@
+# The full story
+
+# Speakers
+## Contact
+
+Ideally, we should start contacting speakers __2.5 months__ prior to the event.
+
+Our tech talks involve a total of __5 long talks__ and __2 lightning__. Someone from our team should always give either one Lightning or a Long talk. Hence, we should make an initial list of _6 people to contact_. If no reply, _follow-up after 3-5 days_ until you get an answer. 
```

Comment:
Like, following up indefinitely until the very end of time? :stuck_out_tongue: 

Shouldn't we stop after the first follow-up if they don't reply? Otherwise it feels kinda spammy.